CONTRIBUTING.md

```diff
@@ -31,6 +31,8 @@ We welcome code contributions through pull requests. Here are some guidelines:
 
 - Open a PR to `main` linking any related issues. Provide detailed context on your changes.
 
+- Add your `OPENAI_API_KEY` environment variable to the repository secrets to make sure your tests can pass.
+

```

Comment:
I think this should be removed as it's a bit confusing, and will likely lead to folks opening Issues or reaching out in the Discrd because they are trying to add their own OpenAI keys to a repo that they don't have access to the Environment Variables for - and I don't think anyone is running the actions on their own forks of the repo.

Reply:
That's fair, I thought it was the reason for the failing tests initially but I realised it wasn't. Will delete.

Would be good to get to the bottom of it more generally since it's a recurring issue..